src/libs/TransactionUtils.js

```diff
@@ -59,14 +59,15 @@ function buildOptimisticTransaction(
     if (originalTransactionID) {
         commentJSON.originalTransactionID = originalTransactionID;
     }
+    const defaultMerchant = _.isEmpty(receipt) ? CONST.TRANSACTION.DEFAULT_MERCHANT : '';
```

Comment:
could you please leave a comment here why it has to be empty?
```suggestion
    // For the SmartScan to run successfully, we need to pass the merchant field empty to the API 
    const defaultMerchant = _.isEmpty(receipt) ? CONST.TRANSACTION.DEFAULT_MERCHANT : '';
```